seq2seq/dataset/vocabulary.py

```diff
@@ -165,7 +173,13 @@ def load(cls, file_name):
             Vocabulary: loaded Vocabulary
         """
         with open(file_name, "rb") as f:
-            return pickle.load(f)
+            size_details = f.readline()
+            sequence = f.readlines()
+        size = int(size_details.replace("size is","").strip())
+        vocab = Vocabulary(size)
+        sequence = [token.strip() for token in sequence]
+        vocab.add_sequence(sequence)
```

Comment:
The meaning of 'sequence', 'token' here is strange.  Why not using `vocab.add_token()` method?